runtime/cudaq/platform/default/rest/helpers/iqm/iqm.config

```diff
@@ -0,0 +1,40 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+
+# Tell DefaultQuantumPlatform what QPU subtype to use
+PLATFORM_QPU=remote_rest
+
+# Tell NVQ++ to generate glue code to set the target backend name
+GEN_TARGET_BACKEND=true
+
+# Add the rest-qpu library to the link list
+LINKLIBS="${LINKLIBS} -lcudaq-rest-qpu"
+
+# Define the lowering pipeline, here we lower to Base QIR
+PLATFORM_LOWERING_CONFIG="expand-measurements,canonicalize,cc-loop-unroll{signal-failure-if-any-loop-cannot-be-completely-unrolled=true},canonicalize,func.func(lower-to-cfg),canonicalize,iqm-gate-set-mapping"
```

Comment:
You can now replace `canonicalize,cc-loop-unroll{signal-failure-if-any-loop-cannot-be-completely-unrolled=true},canonicalize` with `unrolling-pipeline`.

Also, before the final native gate set mapping, you might want to add `func.func(quake-multicontrol-decomposition)`

Reply:
Updated.